### DISCUSSION TITLE
Breakpoint labels

### DISCUSSION THREAD

**Discussion by descawed** [grumpycoders/pcsx-redux#1502] [Ideas]
Sometimes, during complex reverse engineering sessions, I end up with a lot of breakpoints and I have a hard time remembering which one is which. I've thought several times that it would be nice to be able to put a name on breakpoints to remember what they are. I guess I could keep a list in a text editor off to the side or something, but it would be nice to have all the breakpoint info together in the UI. I'd be willing to implement it myself if others think it sounds worthwhile.

**Comment by nicolasnoble** [comment]
That's a very reasonable request, yes. The exact implementation details might be a bit tricky, but that'd start with a

```c++
mutable std::string m_label
```

somewhere in there: https://github.com/grumpycoders/pcsx-redux/blob/19317d85eaa9b9e3ff74ff7a9cc8a6d32579b436/src/core/debug.h#L89-L93
